README.md

```diff
@@ -26,6 +26,15 @@ If you'd like your tests to run automatically when they change, we recommend the
 ```bash
 find . -name "*_shpec.sh" | entr shpec
 ```
+### Test structure
```

Comment:
Mmmmmm, don't know. Sure *test structure* doesn't sound that well. But I'm not that convinced either by *test syntax*